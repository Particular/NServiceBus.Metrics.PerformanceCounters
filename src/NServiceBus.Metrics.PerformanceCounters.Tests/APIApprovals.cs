using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(PerformanceCountersFeature).Assembly);
        Approver.Verify(publicApi);
    }
}