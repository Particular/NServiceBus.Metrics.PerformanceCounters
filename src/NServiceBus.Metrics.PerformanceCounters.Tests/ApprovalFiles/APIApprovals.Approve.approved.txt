[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute(@"NServiceBus.Metrics.PerformanceCounters.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100D32BC524DCB1205998C155A4F36BF873587D3602822ECD7B49CD775B2E6A006EE6B9164AB2E3103A6A4D1310C6E5C26818A32FE86710141A2D1F02EB564381CD64C88131BCCA478CDB5072F06DB991DE33DAC1C82BAF40D9F61DD6B40300A4673B693B51CD10A8B9B7D8AB64450431FA422514D6DABCAF70DF785B1E4E6E8AAF")]
[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
namespace NServiceBus.Metrics.PerformanceCounters
{
    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly | System.AttributeTargets.All)]
    [System.ObsoleteAttribute(@"The NServiceBus.Metrics.PerformanceCounters.MsBuild package is deprecated. There is no longer any need to configure its output. To promote the script outside the build directory, use a post-build event. Will be treated as an error from version 4.0.0. Will be removed in version 5.0.0.", false)]
    public sealed class PerformanceCounterSettingsAttribute : System.Attribute
    {
        public PerformanceCounterSettingsAttribute() { }
        public bool CSharp { get; set; }
        public bool Powershell { get; set; }
        public string ScriptPromotionPath { get; set; }
    }
}
namespace NServiceBus
{
    public class static PerformanceCountersExtensions
    {
        public static NServiceBus.PerformanceCountersSettings EnableWindowsPerformanceCounters(this NServiceBus.EndpointConfiguration endpointConfiguration) { }
    }
    public class PerformanceCountersSettings
    {
        public void EnableSLAPerformanceCounters(System.TimeSpan sla) { }
    }
}